src/hdr-config.cpp

```diff
@@ -504,15 +502,29 @@ namespace librealsense
 
     void hdr_config::set_exposure(float value)
     {
-        _hdr_sequence_params[_current_hdr_sequence_index]._exposure = value;
-        _has_config_changed = true;
+        std::vector<byte> res;
+        _is_enabled = is_hdr_enabled_in_device(res);
+        if (!_is_enabled)
+            _hdr_sequence_params[_current_hdr_sequence_index]._exposure = value;
+        else
+            /*throw wrong_api_call_sequence_exception(
+                rsutils::string::from() << "hdr_config::set_exposure(...) failed! Cannot update HDR when it is already enabled." );*/
+            LOG_WARNING("hdr_config::set_exposure(...) failed! Cannot update HDR when it is already enabled.\n"); 
```

Comment:
For now, WARNING message is printed. Will convert it such that it throws exception.